platform/android/export/export.cpp

```diff
@@ -683,6 +683,22 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 		return enabled_plugins;
 	}
 
+    static Error store_in_gradle_project(const String &p_path, const Vector<uint8_t> &p_data, int compression_method = Z_DEFLATED) {
+        //use dir_access.cpp and file_access.cpp to write the file
+        size_t dir_point = p_path.rfind("/");
```

Comment:
does this work on windows where path slashes are usually "\"?

Reply:
I think there's a method called "fix_path" that I need to call somewhere which takes care of converting paths from '\\' to '/'.

Reply:
Sounds good, thanks.